File: 8kyu 3/removeChar.js
function removeChar(str) {
   const array = str.split("");
   const result = [];
   for (i = 0; i < array.length; i++) {
      if (i != 0 && i != (array.length - 1)) {
         result.push(array[i]);
      }
   }
   return result.join("");
};


function removeChar(str) {
   return str.slice(1, -1);
}


const removeChar = str => str.slice(1, -1)


console.log(removeChar('eloquent'), 'loquen');
console.log(removeChar('country'), 'ountr');
console.log(removeChar('person'), 'erso');
console.log(removeChar('place'), 'lac');

File: Python/8py 3/remove_char.py
"""
|--------------------------------------------------------------------------
| Remove First and Last Character
|--------------------------------------------------------------------------
|
"""

def remove_char(s):
   temp = s.replace(s[0], "", 1)
   return temp[:len(temp)-1]


remove_char = lambda x:x[not():-(not())]


def remove_char(s):
   return s[1 : -1]


def remove_char(s):
   s = list(s)
   s.pop()
   s.pop(0)
   return ''.join(s)


def remove_char(s):
   return s.replace(s[0:],s[1:-1]) 


print(remove_char('eloquent'), 'loquen')
print(remove_char('country'), 'ountr')
print(remove_char('person'), 'erso')
print(remove_char('place'), 'lac')
print(remove_char('ok'), '')
print(remove_char('ooopsss'), 'oopss')
print(remove_char('olcsrs'), 'lcsr')